pyaedt/emit.py

```diff
@@ -100,6 +172,13 @@ def __init__(
         port=0,
         aedt_process_id=None,
     ):
+        if(projectname == None):
+            projectname = generate_unique_project_name()
+        self._emit_api = EmitApiPython.EmitApi()
+        self._result_loaded = False
+        self.location =""
+        self.revisions_list = []
+        self.curr_design = 0
         """Constructor."""
```

Comment:
Is a blank line needed before line 182?

Reply:
I didn't think so because it's not a new method, it's just another single (very long) line of code, let me know if you disagree!